flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/KinesisConfigConstants.java

```diff
@@ -68,10 +97,30 @@
 	//  Default configuration values
 	// ------------------------------------------------------------------------
 
-	public static final int DEFAULT_STREAM_DESCRIBE_RETRY_TIMES = 3;
+	public static final long DEFAULT_STREAM_DESCRIBE_BACKOFF_BASE = 1000L;
+
+	public static final long DEFAULT_STREAM_DESCRIBE_BACKOFF_MAX = 5000L;
+
+	public static final double DEFAULT_STREAM_DESCRIBE_BACKOFF_EXPONENTIAL_CONSTANT = 1.5;
+
+	public static final int DEFAULT_SHARD_GETRECORDS_MAX = 100;
+
+	public static final int DEFAULT_SHARD_GETRECORDS_RETRIES = 3;
+
+	public static final long DEFAULT_SHARD_GETRECORDS_BACKOFF_BASE = 300L;
+
+	public static final long DEFAULT_SHARD_GETRECORDS_BACKOFF_MAX = 1000L;
+
+	public static final double DEFAULT_SHARD_GETRECORDS_BACKOFF_EXPONENTIAL_CONSTANT = 1.5;
+
+	public static final int DEFAULT_SHARD_GETITERATOR_RETRIES = 3;
+
+	public static final long DEFAULT_SHARD_GETITERATOR_BACKOFF_BASE = 300L;
+
+	public static final long DEFAULT_SHARD_GETITERATOR_BACKOFF_MAX = 1000L;
 
-	public static final long DEFAULT_STREAM_DESCRIBE_BACKOFF = 1000L;
+	public static final double DEFAULT_SHARD_GETITERATOR_BACKOFF_EXPONENTIAL_CONSTANT = 1.5;
 
-	public static final int DEFAULT_SHARD_RECORDS_PER_GET = 100;
+	public static final long DEFAULT_SHARD_DISCOVERY_INTERVAL_MILLIS = 10000L;
```

Comment:
I'm using 10s for default discovery interval here. I tested it and the originally suggested 30s seemed a bit too long as a default, IMHO. Can change it back to 30s if you think it's more appropriate.